pkg/client/cache/expiration_cache.go

```diff
@@ -146,6 +146,10 @@ func (c *ExpirationCache) ListKeys() []string {
 func (c *ExpirationCache) Add(obj interface{}) error {
 	c.expirationLock.Lock()
 	defer c.expirationLock.Unlock()
+	return c.addLocked(obj)
+}
+
+func (c *ExpirationCache) addLocked(obj interface{}) error {
```

Comment:
Following the naming convention I see for similar function (like in delta_fifo), it should be addLocked